internal/api/pacts/pull_request_review_submitted_approval_mode_require_any.json

```diff
@@ -79,7 +79,7 @@
         "headers": {
           "Content-Type": "text/plain; charset=utf-8"
         },
-        "body": "pull_request_approval_rules:\n- target_branches:\n  - master\n  rules:\n  - approval_mode: require_any\n    regex: \"- \\\\[x\\\\] Yes - this change impacts customers\"\n    approving_team_handles:\n    -  CAB - Foo\n    -  CAB - Bar\n    labels:\n    - needs-cab-approval\n  - regex: \"- \\\\[x\\\\] Yes - this change impacts documentation\"\n    approving_team_handles:\n    -  CAB - Documentation\n    labels:\n    - needs-doc-approval\n"
+        "body": "pull_request_approval_rules:\n- target_branches:\n  - master\n  rules:\n  - approval_mode: require_any\n    regex: \"- \\\\[x\\\\] Yes - this change impacts customers\"\n    approving_team_handles:\n    -  CAB - Foo\n    -  CAB - Bar\n    labels:\n    - needs-cab-approval\n  - approval_mode: require_all\n    regex: \"- \\\\[x\\\\] Yes - this change impacts documentation\"\n    approving_team_handles:\n    -  CAB - Documentation\n    labels:\n    - needs-doc-approval\n"
```

Comment:
I am confused by setting `approval_mode: require_all`. Pact name `internal/api/pacts/pull_request_review_submitted_approval_mode_require_any.json` indicates mode should be `require_any`, no?

Reply:
It should. It was a `require_all` with one approving team handle anyway. Copy-paste mistake.